src/js/media/controllers/replace-image.js

```diff
@@ -90,24 +90,28 @@ ReplaceImage = Library.extend(/** @lends wp.media.controller.ReplaceImage.protot
 	 * @since 3.9.0
 	 */
 	activate: function() {
-		this.updateSelection();
+		this.frame.on( 'content:render:browse', this.updateSelection, this );
+
 		Library.prototype.activate.apply( this, arguments );
 	},
 
+	/**
+	 * @since 5.9.0
+	 */
+	deactivate: function() {
```

Comment:
Hi @hellofromtonya ! Is this function still needed in this PR? I'm not seeing where it is being used. Maybe it is part of another PR. :) If you can confirm this function, the rest is good to go in my testing.

Reply:
@anthonyburchell See comment https://core.trac.wordpress.org/ticket/53765#comment:51

Reply:
@anthonyburchell Asked why this is here. Here's the answer https://core.trac.wordpress.org/ticket/53765#comment:69.